tools/ebpf/Dockerfiles/Dockerfile-process-agent-dev

```diff
@@ -13,8 +13,13 @@ ENV LD_LIBRARY_PATH /opt/datadog-agent/embedded/lib
 COPY process-agent /opt/datadog-agent/embedded/bin/process-agent
 COPY system-probe /opt/datadog-agent/embedded/bin/system-probe
 
-# comment out the below if you don't want the replace the base image's core agent binary
-COPY agent /opt/datadog-agent/bin/agent/agent
```

Comment:
One way we could simplify this, is to have the destination path be a build arg (https://docs.docker.com/engine/reference/builder/#arg). It could default to `/opt/datadog-agent/bin/agent/agent`, but if you didn't want to overwrite it, you could set the arg from the invoke task using `--build-arg agentdest=/dev/null`. (I think using `/dev/null` here would work). This would require the copy of `agent` to the docker context not be optional though.

Reply:
Good idea, I'll update this